### ISSUE TITLE
add favicon.ico

### ISSUE THREAD

**Issue by szabgab** [kantord/LibreLingo#1791]
Currently https://librelingo.app/favicon.ico return 404
I saw #1001 but even before that gets ready the current image https://librelingo.app/images/logo.svg could be used as favicon.

**Comment by kantord** [comment]
Good idea. Perhaps it could be simplified using a[ webpack plugin](https://www.npmjs.com/package/favicons-webpack-plugin) such as this one. 

I'd say **acceptance criteria** should be that it also support modern favicon alternatives, like the plugin above does.

**Comment by kantord** [comment]
@allcontributors please add @szabgab  for ideas
